src/ctap2/mod.rs

```diff
@@ -1298,12 +1432,20 @@ pub(crate) fn configure_authenticator(
         let resp = dev.send_cbor_cancellable(&authcfg, alive);
         match resp {
             Ok(()) => {
-                callback.call(Ok(ManageResult::Success));
-                break;
+                let auth_info = unwrap_option!(dev.refresh_authenticator_info(), callback);
+                send_status(
+                    &status,
+                    StatusUpdate::InteractiveManagement(InteractiveUpdate::AuthConfigUpdate((
+                        AuthConfigResult::Success(auth_info.clone()),
+                        Some(pin_uv_auth_result),
+                    ))),
+                );
+                return true;
```

Comment:
And success here?

Reply:
No, that would close the interactive channel on first success. We inform the user about successes via `StatusUpdate`.
There currently is actually no pathway to trigger a `callback.call(Ok(ManageResult::Success));` here.
The only way to "end the call" is letting the user "hang up", making us exit with `CancelledByUser`.
Maybe we could think about introducing an `InteractiveRequest::Quit`-variant that would then trigger the Ok-callback, if we want this?

Reply:
I've implemented this now.